.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "data"]
 	path = data
 	url = https://github.com/ORNL-CEES/DTKData.git
+[submodule "arborx"]
+	path = packages/Search
+	url = https://github.com/arborx/arborx.git
```

Comment:
Not case sensitive?

Reply:
http url are not case sensitive.

Reply:
There are though. Only the domain name is not case sensitive https://stackoverflow.com/questions/7996919/should-url-be-case-sensitive

Reply:
FWIW I did a clone recursive of Andrey's branch and it worked

Reply:
@Rombur Right you are.

That being said, github *is* case-insensitive for repos and organizations (though I have not found the exact place on GitHub stating that), so it's not a big deal.